### DISCUSSION TITLE
RandCropByPosNegLabeld crop position

### DISCUSSION THREAD

**Discussion by sharonlee12** [Project-MONAI/MONAI#7761] [General]
Hello, I would like to know if the  RandCropByPosNegLabel cropped position  can be returned? Or is there a transform that makes it easier to obtain the cropping position?

**Comment by KumoLiu** [comment]
Hi @sharonlee12, thanks for your interest here.
You can get crop center by using `data.meta["crop_center"]`.
https://github.com/Project-MONAI/MONAI/blob/4af23069ccbbd26d7816bafc3762365ce4c5da77/monai/transforms/croppad/array.py#L1220

Hope it helps, thanks. (convert to discussion for now)
